identity-iota/src/tangle/client_builder.rs

```diff
@@ -29,6 +30,7 @@ impl ClientBuilder {
       network: Default::default(),
       builder: iota_client::ClientBuilder::new().with_local_pow(DEFAULT_LOCAL_POW),
       encoding: DIDMessageEncoding::JsonBrotli,
+      retry_until_included: true,
```

Comment:
Not sure if we should follow `DEFAULT_LOCAL_POW` and make the default a constant too?

```suggestion
      builder: iota_client::ClientBuilder::new().with_local_pow(DEFAULT_LOCAL_POW),
      encoding: DIDMessageEncoding::JsonBrotli,
      retry_until_included: DEFAULT_RETRY_UNTIL_INCLUDED,
```